app/templates/login.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>DMP Profiler Demo</title>
+    <link rel="shortcut icon" href="{{url_for('cprofiler.static', filename='favicon.ico')}}" type="image/x-icon">
+    <!-- <link rel="icon" href="/favicon.ico" type="image/x-icon"> -->
+    <link rel="stylesheet" type="text/css" href="static/login.css">
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script>
+      function login(){
+          $.ajax({
+            url: "http://127.0.0.1:5000/cprofiler/login",
+            type: "POST",
+            dataType: "json",
+            data: {
+                username: $("#username").val(),
+                password: $("#password").val()
+            }
+          }).then(data => console.log('data : ', data), 
```

Comment:
@billydentsu 
end login in amjiltai bolson ued, index html-ee access tokentei hamt duudah yostoi bdag uu?
Medeh uu?

Reply:
@od-erdene 
Sain medehguiee, Inhee-gees asuuchih